addons/turbowarp-player/userscript.js

```diff
@@ -27,12 +27,22 @@ export default async function ({ addon, console, msg }) {
   }
 
   button.onclick = async () => {
+    const projectId = window.location.pathname.split("/")[2];
     let search = "";
+    let projectToken = (
+      await (
+        await fetch(`https://api.scratch.mit.edu/projects/${projectId}&nocache=${Date.now()}`, {
+          headers: {
+            "x-token": await addon.auth.fetchXToken(),
```

Comment:
Scratch returns a 401 error when signed out

Reply:
There's no way of getting the project if you're signed out, because you can't access the token. Should we just disable the button when you're signed out?

Reply:
The project API is accessible on shared projects while signed out.

Reply:
Alright, I fixed it to only include the headers when signed in.